webapp/security/views.py

```diff
@@ -263,8 +273,107 @@ def api_create_notice():
 # CVE views
 # ===
 def cve_index():
-    return flask.render_template("security/cve/index.html")
+
+    # Temporary fix for data to be displayed
+    cves_query = db_session.query(CVE).first()
+    if not cves_query and os.environ["FLASK_DEBUG"] == "true":
+        api_create_cve()
+    list_cve = cves_query.order_by(CVE.public_date).limit(10).all()
+
+    return flask.render_template("security/cve/index.html", list_cve=list_cve)
 
 
 def cve(cve_id):
-    return flask.render_template("security/cve/cve.html")
+
+    # Temporary fix for data to be displayed
+    cves_query = db_session.query(CVE).first()
+    if not cves_query and os.environ["FLASK_DEBUG"] == "true":
+        api_create_cve()
+
+    cve = db_session.query(CVE).get(cve_id.upper())
+    if not cve:
+        flask.abort(404)
+    return flask.render_template("security/cve/cve.html", cve=cve)
+
+
+# CVE API
+# ===
+
+
+def api_create_cve():
```

Comment:
This is a views file and this looks more like an helper function to load things into the DB. This is probably what your helper script looks like.